frontend/src/components/CommentsOnPost.jsx

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from 'react';
+import 'semantic-ui-css/semantic.min.css';
+import { Comment, Header, Form, Input, Button, Loader, Message} from 'semantic-ui-react';
+import ProfileBubble from './ProfileBubble';
+import Cookies from 'js-cookie';
+import HTTPFetchUtil from "../util/HTTPFetchUtil";
+import PropTypes from 'prop-types';
+import './styles/CommentsOnPost.css';
+
+class CommentsOnPost extends Component {	
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			comments: [],
```

Comment:
this file is part of your old comments PR?

Reply:
o oops, I'll take this file and the css file out.